include/deal.II/base/std_cxx11/type_traits.h

```diff
@@ -77,6 +79,9 @@ namespace std_cxx11
                               boost::has_trivial_constructor<T>::value &&
                               boost::has_trivial_destructor<T>::value;
   };
+
+  using boost::true_type;
+  using boost::false_type;
```

Comment:
[`true_type`](https://github.com/dealii/dealii/blob/master/bundled/boost-1.62.0/include/boost/type_traits/integral_constant.hpp#L101) and [`false_type`](https://github.com/dealii/dealii/blob/master/bundled/boost-1.62.0/include/boost/type_traits/integral_constant.hpp#L102) are definitely defined in  the`integral_constant` header, and in the bundled boost this is included in [`boost/type_traits.hpp`](https://github.com/dealii/dealii/blob/master/bundled/boost-1.62.0/include/boost/type_traits.hpp#L84).